src/Controllers/UploadController.php

```
@@ -255,10 +255,35 @@ public function generateSummary(): void
             return;
         }
 
+        if ($result['status'] === 'policy_denied') {
+            $reason = $this->summaryPolicyDeniedReason((string) ($result['reason'] ?? 'policy_denied'));
+            \StratFlow\Services\Logger::warn('[DocumentSummary] policy denied reason=' . $reason);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Services\Logger' in method 'generateSummary'. (undefined)

(StaticAccess)